### PR TITLE
Add support for testing bad-words file

### PR DESCRIPTION
**Pull request by pradh**
Instructions in cl/651193865

### CODE REVIEW

File: run_test.sh [hunk 1]
```diff
@@ -168,7 +168,11 @@ function run_integration_test {
   export FLASK_ENV=integration_test
   export DC_API_KEY=
   export LLM_API_KEY=
-  export ENV_PREFIX=Staging
+  if [[ $ENV_PREFIX == "" ]]; then
+    export ENV_PREFIX=Staging
+  else
+    echo "Using ENV_PREFIX=$ENV_PREFIX"
```
Comment (chejennifer) [inline run_test.sh:174]: nit: maybe can always echo this even when the script manually sets the ENV_PREFIX to staging

File: run_test.sh [hunk 2]
```diff
@@ -191,7 +195,11 @@ function update_integration_test_golden {
   python3 -m pytest -vv --reruns 2 server/integration_tests/topic_cache
 
   # Run integration test against staging mixer to make it stable.
-  export ENV_PREFIX=Staging
+  if [[ $ENV_PREFIX == "" ]]; then
+    export ENV_PREFIX=Staging
+  else
+    echo "Using ENV_PREFIX=$ENV_PREFIX"
```
Comment (chejennifer) [inline run_test.sh:201]: also here, maybe can always echo this line

**Review by chejennifer — APPROVED** [review]
nice!

File: run_test.sh (hunk 1 above)
Comment (pradh) [reply run_test.sh:174]: Done.

File: run_test.sh (hunk 2 above)
Comment (pradh) [reply run_test.sh:201]: Done.

**Pull request merged**